library/spdm_responder_lib/libspdm_rsp_algorithms.c

```diff
@@ -683,16 +699,6 @@ libspdm_return_t libspdm_get_response_algorithms(libspdm_context_t *spdm_context
                     SPDM_ERROR_CODE_INVALID_REQUEST, 0,
                     response_size, response);
             }
-            if (spdm_response->header.spdm_version >= SPDM_MESSAGE_VERSION_12) {
-                if ((spdm_context->connection_info.algorithm.other_params_support &
-                     SPDM_ALGORITHMS_OPAQUE_DATA_FORMAT_MASK) !=
-                    SPDM_ALGORITHMS_OPAQUE_DATA_FORMAT_1) {
-                    return libspdm_generate_error_response(
-                        spdm_context,
-                        SPDM_ERROR_CODE_INVALID_REQUEST, 0,
-                        response_size, response);
-                }
-            }
```

Comment:
Should we revert this as well?

Reply:
Sure. Long-term I don't think the Responder should return the error.